manzi-guev/Broadcaster
# #169921362 User sign up endpoint

### What does this PR do?
- It creates the endpoint for a user to login
### Description of the task to be completed?
- Writing the user login endpoint
- Writing the tests for the endpoint
### How should this be manually tested?
- Make sure you have Postman running
- Run the project using 
```
npm dev
```
- After running, go to Postman and provide the route then send the request
### What are the relevant pivotal tracker stories?
[#169921362](https://www.pivotaltracker.com/story/show/169921362)
### Screenshots
![image](https://user-images.githubusercontent.com/53451069/69474665-74114600-0dcc-11ea-894a-f60e26d52b36.png)
![image](https://user-images.githubusercontent.com/53451069/69474689-b8044b00-0dcc-11ea-92f2-6c51e7d35f7a.png)
